### ISSUE TITLE
Support siblings

### ISSUE THREAD

**Issue by stereodenis** [hiukky/eslint-plugin-hooks-sort#1]
We need support of siblings hooks, that can be in any order to each other

```
'hooks-sort/order-hooks': [
      2,
      {
        groups: [
          ['useNavigation', 'useLanguage', 'useTranslation'],
          'useRef',
          ['useIsKeyboardShown', 'useWindowDimensions'],
          'useForm',
          'useState',
          ['useQuery', 'useLazyQuery', 'useMutation'],
          'useToggle',
          ['useCallback', 'useMemo'],
          'useEffect',
        ],
      },
    ],
```

Like it did in import/order
```
'import/order': [
      'error',
      {
        groups: ['external', 'internal', 'builtin', ['sibling', 'parent'], 'index'],
        'newlines-between': 'always',
      },
    ],
```

**Comment by hiukky** [comment]
Hi @stereodenis

First thanks for your contribution! Could you give me a code example of the result you expect? at the moment I have a little time to work on this feat but a PR is welcome.

I will take a look at this import plugin and see what result I can get, I don't know if this behavior is interesting, but depending on its reproduction in code I can have ideas for this implementation.

**Comment by stereodenis** [comment]
I expect, that if I write like in the example above, there will be not important, what will be first: useCallback or useMemo

**Comment by hiukky** [comment]
@stereodenis  I'm sorry, I still haven't figured out what to do. If you can provide a snippet of codes exemplifying the problem that should be solved it would be great. so I can see if it really is something viable to do and if it does not escape the focus of the plugin. : )

**Comment by hiukky** [comment]
Problem solved in version 0.1.0! Thanks.

**Comment by hiukky** [comment]
Feel free to reopen this problem if you can be more descriptive in relation to the problem or if you have any suggestions or PR! Thanks!
